cloud/services/groups/groups.go

```diff
@@ -34,7 +33,7 @@ func (s *Service) Reconcile(ctx context.Context, spec interface{}) error {
 		// resource group already exists, skip creation
 		return nil
 	}
-	s.Scope.Logger.V(2).Info("creating resource group", "resource group", s.Scope.ResourceGroup())
+	s.Scope.V(2).Info("creating resource group", "resource group", s.Scope.ResourceGroup())
```

Comment:
this is going to conflict with your groups PR

Reply:
no problem we rebase :)